### ISSUE TITLE
Arabic OCR Training problem

### ISSUE THREAD

**Issue by Dawood180** [JaidedAI/EasyOCR#1423]
Hello, I am currently trying to improve easyOCR recognition model by training same arabic and english characters with extra symbols added to the pretrained model. 
By using the pretrained arabic model, the model forgets the labels as the weights of the output layer is randomly initialized. Is there a way to load pretrained model with same weights in output layer and random initialize weights of extra symbols of output layer?

Also, what is the best config to be used during training to get better results? I have tried Ada delta, Adam and SGD. SGD performs best results so far while Ada delta and Adam performed poor results, what are the reasons? I saw other issues where some people said that training using Ada delta had good results.

The training data consists of 1.3M images including augmentations. Is that enough or do I need to add more training data?

Here is the config file I am using for training:
[config.txt](https://github.com/user-attachments/files/21212267/config.txt)

**Comment by Basel-Mohamed** [comment]
any update ?
